python/activity_stream/activity_stream.py

```diff
@@ -150,6 +158,29 @@ def destroy(self):
     # properties
 
     @property
+    def note_threads(self):
+        """
+        The currently loaded note threads, keyed by Note entity id and
+        containing a list of Shotgun entity dictionaries.
+
+        Example structure containing a single Note entity:
+            6038: [
+                {
+                    'content': 'This is a test note.',
```

Comment:
is the first item always a Note entity and then there are reply entities following? Could there be attachments in this list?